### DISCUSSION TITLE
Arbitrary protein structure prediction

### DISCUSSION THREAD

**Discussion by tokiinan5** [Urinx/alphafold_pytorch#20] [Ideas]
The original Alpha fold does not work for any arbitrary protein- it only works for CASP-13. I want to make it work for any sequence of amino acid. Any suggestion?

**Comment by Planet-KIM** [comment]
Me too. I want use any sequence of amino acid.
How can I get the pkl and tfrec files from the pdb file?
- I want to use "SARS-CoV-2" protein. Give me the solutions.
